lib/beaker/host_prebuilt_steps.rb

```diff
@@ -12,7 +12,8 @@ module HostPrebuiltSteps
     NTPSERVER = 'pool.ntp.org'
     SLEEPWAIT = 5
     TRIES = 5
-    RHEL8_PACKAGES = ['curl', 'chrony']
+    AMAZON2023_PACKAGES = %w[curl-minimal chrony]
+    RHEL8_PACKAGES = %w[curl chrony]
```

Comment:
Nit: would be nice to also separate this change. A bit further down there are more `[]` to `%w[]` changes that are unrelated.

Reply:
Hi @ekohl,
Yes, this hierarchy suggests that amazon facts are going to be derived from the redhat family.
https://github.com/puppetlabs/facter/blob/83ef0399cece0b475a97e04e2743561d703d7ca3/lib/facter/config.rb#L20C1-L22C21